test/f32-rwdsum.yaml

```diff
@@ -0,0 +1,7 @@
+# Copyright 2024 Google LLC
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# SCALAR
+- name: xnn_f32_rwdsum_ukernel_1p1x__scalar_c1
```

Comment:
Please don't add new yaml files, and instead try to use this new system: https://github.com/google/XNNPACK/blob/master/doc/microkernel-enumerators.md, e.g. https://github.com/google/XNNPACK/blob/master/src/f16-vabs/f16-vabs.h.

We are working on converting existing yamls to such headers.